pkg/pb/orders.proto

```diff
@@ -60,6 +60,10 @@ message PieceHash {
     bytes hash = 2;
     // signature either satellite or storage node
     bytes signature = 3;
+    // size of uploaded piece
+    int64 piece_size = 4;
+    // timestamp when upload occur
+    google.protobuf.Timestamp timestamp = 5 [(gogoproto.stdtime) = true];
```

Comment:
Oh, I've just learnt this can be actually annotated with `[(gogoproto.stdtime) = true, (gogoproto.nullable) = false]`, which will make `timestamp` of type `time.Time` instead of `*time.Time`. This will resolve the issue in some places to get a pointer to `time.Now()`.

Reply:
even better, nice!